src/handwritten/task.py

```diff
@@ -405,34 +405,31 @@ def is_task_done(self):
 
     def perform_strain_shunt_cal(
             self, channel="", shunt_resistor_value=10000,
-            shunt_resistor_location=0, shunt_resistor_select=0,
-            shunt_resistor_source=0, skip_unsupported_channels=False):
+            shunt_resistor_location=ShuntElementLocation.R1, shunt_resistor_select=ShuntCalSelect.A,
+            shunt_resistor_source=ShuntCalSource.BUILT_IN, skip_unsupported_channels=False):
         """
         Performs shunt calibration for the specified channels of the task. 
-        The instances of this polymorphic VI correspond to the type of bridge 
+        The instances of this method correspond to the type of bridge 
         sensor. Refer to the calibration procedure for your module for detailed
         calibration instructions.
```

Comment:
On further thought, it looks like you pulled the 1st and 2nd paragraphs from the poly VI help and the instance VI help, respectively. For Python, it would make sense to combine them.

Also: the 1st line of a Python docstring should be a one-line summary. It should not be part of a larger paragraph.

How about this? (If the 1st line fits in 100 chars)

> Performs shunt calibration for the specified channels using a strain gage sensor.
> 
> Refer to the calibration procedure for your module for detailed calibration instructions.

Ditto for the other method.

Reply:
The summary line should be a command ("Perform" not "Performs") as per https://peps.python.org/pep-0257/#one-line-docstrings